apps/root/src/pages/aggregator/swap-container/components/swap-recap-data/index.tsx

```diff
@@ -0,0 +1,94 @@
+import TokenIcon from '@common/components/token-icon';
+import { formatCurrencyAmount, getNetworkCurrencyTokens } from '@common/utils/currency';
+import { NETWORKS } from '@constants';
+import { useAggregatorState } from '@state/aggregator/hooks';
+import { useThemeMode } from '@state/config/hooks';
+import { find } from 'lodash';
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import styled from 'styled-components';
+import { ContainerBox, Divider, EastIcon, Typography, colors } from 'ui-library';
+
+const RecapDataContainer = styled(ContainerBox).attrs({ flexDirection: 'column', alignItems: 'start' })``;
+
+const SwapRecapData = () => {
+  const { network: chainId, selectedRoute } = useAggregatorState();
+  const themeMode = useThemeMode();
+
+  const network = find(NETWORKS, { chainId });
+  if (!network || !selectedRoute) {
+    return null;
+  }
+
+  const { nativeCurrencyToken } = getNetworkCurrencyTokens(network);
+
+  return (
+    <ContainerBox gap={8}>
+      <ContainerBox gap={3} alignItems="center">
+        <RecapDataContainer>
+          <Typography variant="bodySmall">
+            <FormattedMessage description="youPay" defaultMessage="You pay" />
+          </Typography>
+          <ContainerBox gap={2} alignItems="center">
+            <TokenIcon token={selectedRoute.sellToken} size={5} />
+            <Typography variant="body" fontWeight="bold">
+              {formatCurrencyAmount(selectedRoute.sellAmount.amount, selectedRoute.sellToken, 2)}{' '}
+              {selectedRoute.sellToken.symbol}
+            </Typography>
+          </ContainerBox>
+          <Typography variant="bodySmall" textAlign="center" pl={7}>
```

Comment:
whats the pl prop?

Reply:
Padding left. 5 from `TokenIcon` + 2 from flex `gap` = 7, so it's aligned with amountInUnits value

Reply:
Gotcha, instead of doing that since it could create some issues in some browsers due to how px and gaps and px's are calculated. What about spliting the box like this? So the amountInUnits value will ALWAYS be aligned with the usd amount
![Untitled drawing (7)](https://github.com/Mean-Finance/dca-fe/assets/94012134/4b4e351c-382a-4975-81e6-9baea9d672b2)

Reply:
Oh my... This new box structure almost made it, but It was hard for me to also align the icon with the amount:
![Screenshot 2024-02-08 at 11 35 10](https://github.com/Mean-Finance/dca-fe/assets/144699868/038ef42b-2e24-4bc4-be70-1473afee99cc)

I found a way to achieve this using grid! (I have leveraged Mui Box component, although we were trying to avoid it, but find this so useful: https://mui.com/system/grid/)
<img width="174" alt="Screenshot 2024-02-08 at 15 14 15" src="https://github.com/Mean-Finance/dca-fe/assets/144699868/54852491-a6f9-4161-adc1-0a1affde8f09">

Let me know what you think!